### ISSUE TITLE
historic=memorial and tourism=artwork conflicts

### ISSUE THREAD

**Issue by Janjko** [gravitystorm/openstreetmap-carto#3565]
There is a problem with tags _tourism=artwork_ and _historic=memorial_. They have subtags with the same meaning: artwork has _artwork_type=sculpture;statue;bust_ and memorial has _memorial=sculpture;statue;bust_. A bust icon  ![memorial bust](https://user-images.githubusercontent.com/25656654/36630295-83cc10f6-1964-11e8-8547-276221216235.png) is rendered if you put _historic=memorial+memorial=bust_. But if you add _tourism=artwork_ on the same node, it is rendered like a generic artwork  ![tourism artwork](https://user-images.githubusercontent.com/25656654/44617373-579cfb80-a861-11e8-80d5-e70764044744.png). Even if you put:

_historic=memorial+memorial=bust+tourism=artwork+artwork_type=bust_

it still doesn't render a bust.
In reality, all memorial=bust are also tourism=artwork, so it isn't wrong tagging.

In my opinion we should first decide which tag is stronger in case of different values, _memorial=*_ or _artwork_type=*_. After that, just render whatever is in those tags, or in the stronger tag if there are both.

**Comment by kocio-pl** [comment]
It is documented on the wiki, so I think it's also worth rendering:

https://wiki.openstreetmap.org/wiki/Key:artwork_type

It might be good to also review other types of artwork by the way.

**Comment by jragusa** [comment]
Bust and stone are concerned. I propose to render for now `artwork_type=bust` with the current `memorial=bust` icon. It's a rapid fix. Icon of `memorial=stone` suggests an inscription so I propose for this one to wait for the rendering of natural=stone as node (#2616).

**Comment by dieterdreist** [comment]
sent from a phone

> On 12. Dec 2018, at 21:49, Janjko <notifications@github.com> wrote:
> 
> They have subtags with the same meaning:


can you explain what the problem is?

**Comment by geozeisig** [comment]
Using the icon ![tourism artwork](https://user-images.githubusercontent.com/25656654/44617373-579cfb80-a861-11e8-80d5-e70764044744.png) for an entire building was for me confusing. This happens when `artwork_type=architecture` is applied. An example is the [Siemens-Villa](https://www.openstreetmap.org/way/30595355#map=18/52.43883/13.32918&layers=N). Maybe you should make an exception for `artwork_type=architecture`.

**Comment by jragusa** [comment]
@geozeisig good point. Why not the upper part of a greek column of [Ionic order](https://en.wikipedia.org/wiki/Ionic_order) as icon ?
